src/main/java/frc/robot/SystemFactory.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot;
+
+
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import frc.robot.subsystems.Swerve;
+import frc.robot.subsystems.SwerveModule;
+
+
+public class SystemFactory {
+
+
+    public SystemFactory(){
```

Comment:
```suggestion
    private SystemFactory(){
```
make this private. I mean, this is a helper class, with static methods, so no point having a constructor really. But what one can do is create a `private` constructor to prevent anyone from creating an instance of the class (because they shouldn't)